roles/opensrp/tasks/main.yml

```diff
@@ -63,10 +63,26 @@
     owner: "{{ opensrp_system_user }}"
     group: "{{ tomcat_group }}"
 
+- name: Download CA certificate bundle
+  become: yes
+  become_user: root
+  get_url:
+    url: "{{ ca_bundle_url }}"
+    dest: "/usr/local/share/ca-certificates/ca-bundle.crt"
```

Comment:
@ndegwamartin this isn't a deal breaker, but do you think it'd be a good idea to rename the file `ca-bundle.crt` to something that is more reflective that this is related to OpenSRP and database access to prevent a collision?

Reply:
Yeah that makes sense. so like **_db-ca-bundle.crt_**?

Reply:
Or **_opensrp-db-ca-bundle.crt_** ?

Reply:
I've updated this to be **postgres-ca-bundle.crt** as per our discussion earlier